src-clean/config.js

```diff
@@ -10,6 +10,8 @@ const config = {
     env: process.env.NODE_ENV || 'development',
     baseUrl: `http://${ip}:${port}${basename}`,
     apiUrl: `http://${ip}:${port}/api`,
+    graphqlUrl: `http://${ip}:${port}/api/graphql`,
+    serverGraphqlUrl: 'http://localhost:3000/api/graphql',
```

Comment:
I think we should make `baseUrl` a variable (like `ip` and `port`) and use it for `apiUrl`, `graphqUrl` and `serverGraphqlUrl`. The `basename` is necessary for cases when you need to serve the project in a path other than root (e.g. to serve on `https://diegohaz.github.io/foo` we need to `PUBLIC_PATH=foo npm run build`)